gef.py

```diff
@@ -8086,9 +8035,7 @@ def do_invoke(self, argv: List[str]) -> None:
             bp.delete()
 
         # break at entry point
-        entry = get_entry_point()
-        if entry is None:
-            return
+        entry = gef.binary.entry_point
```

Comment:
are you sure that `gef.binary` can never be `None` here?

Reply:
Yes, `gef.binary` will be set by the new objfile handler is trigger and raise if `ELF` fails to parse the header. So this line cannot be reached unless a file is assigned for the session. Right now we can even simplify the first 2 tests of `EntryPointBreakCommand.do_invoke` by simply one `if not gef.binary: bail`. Todo later